src/modules/cmd_duration.rs

```diff
@@ -51,14 +51,26 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(undistract_me(module, &config, elapsed))
 }
 
+#[cfg(not(feature = "notify"))]
+fn undistract_me<'a, 'b>(
+    module: Module<'a>,
+    _config: &'b CmdDurationConfig,
+    _elapsed: u128,
+) -> Module<'a> {
+    module
+}
+
+#[cfg(feature = "notify")]
 fn undistract_me<'a, 'b>(
     module: Module<'a>,
     config: &'b CmdDurationConfig,
     elapsed: u128,
 ) -> Module<'a> {
     use ansi_term::{unstyle, ANSIStrings};
+    #[cfg(feature = "notify")]
     use notify_rust::{Notification, Timeout};
 
+    #[cfg(feature = "notify")]
```

Comment:
LGTM, but please remove the inner `cfg`s:

```suggestion
    use notify_rust::{Notification, Timeout};

```